ci-operator/step-registry/upi/conf/vsphere/ova/upi-conf-vsphere-ova-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+HOME=/tmp
+export HOME
+
+echo "$(date -u --rfc-3339=seconds) - Locating RHCOS image for release..."
+ova_url="$(jq -r '.baseURI + .images["vmware"].path' /var/lib/openshift-install/rhcos.json)"
+vm_template="${ova_url##*/}"
+
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+mkdir /tmp/rhcos
+cat > /tmp/rhcos/rhcos.json << EOF
+{
+   "DiskProvisioning": "thin",
+   "MarkAsTemplate": false,
+   "PowerOn": false,
+   "InjectOvfEnv": false,
+   "WaitForIP": false,
+   "Name": "${vm_template}"
+}
+EOF
+
+echo "$(date -u --rfc-3339=seconds) - Checking if RHCOS OVA needs to be downloaded from ${ova_url}..."
```

Comment:
Might be worth teaching ci-operator to pull down timestamped container logs so we don't need to work up one-off manual injection like this.  @stevekuznetsov, thoughts?